.github/workflows/build_main.yml

```diff
@@ -0,0 +1,18 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - feature/main-builds
```

Comment:
is this branch protected too? Would we want that?

Reply:
I need to remove this feature branch from that. I just had that for testing